account/account.go

```diff
@@ -824,3 +824,49 @@ func FmtCalldataCairo2(fnCalls []rpc.FunctionCall) []*felt.Felt {
 
 	return execCallData
 }
+
+type DeployOptions struct {
+	ClassHash           *felt.Felt
+	MaxFee              *felt.Felt
+	DeploytWaitTime     time.Duration
+	ConstructorCalldata []*felt.Felt
+}
```

Comment:
Can you add comments (eg Classhash, ConstructorCallData). There's also a typo in DeploytWaitTime.